Assets/Scripts/Models/NeedActions.cs

```diff
@@ -13,43 +13,18 @@
 
 public class NeedActions
 {
-    private static NeedActions instance;
-
-    private Script myLuaScript;
-
-    private NeedActions()
+    static NeedActions()
     {
-        // Tell the LUA interpreter system to load all the classes
-        // that we have marked as [MoonSharpUserData]
-        UserData.RegisterAssembly();
-
-        myLuaScript = new Script();
-
-        // If we want to be able to instantiate a new object of a class
-        //   i.e. by doing    SomeClass.__new()
-        // We need to make the base type visible.
-        myLuaScript.Globals["Inventory"] = typeof(Inventory);
-        myLuaScript.Globals["Job"] = typeof(Job);
-        myLuaScript.Globals["ModUtils"] = typeof(ModUtils);
-
-        // Also to access statics/globals
-        myLuaScript.Globals["World"] = typeof(World);
-
-        LoadScripts();
+        LoadScript();
+        LoadModsScripts(WorldController.Instance.modsManager.GetMods());
```

Comment:
LoadModScripts causes the NRE, mod handling is already handled through ModManager.